src/core_plugins/kibana/public/context/api/utils/sorting.js

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+
+
+function reverseQuerySort(query) {
+  return {
+    ...query,
+    sort: _.get(query, 'sort', []).map(reverseSortDirective),
+  };
+}
+
+function reverseSortDirective(sortDirective) {
+  if (_.isString(sortDirective)) {
+    return {
+      [sortDirective]: (sortDirective === '_score' ? 'asc' : 'desc'),
```

Comment:
Just following the elasticsearch defaults: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-sort.html#_sort_order. The documentation does not state any reason.